.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 dist: trusty
 language: scala
 scala:
- - 2.11.12
+ - 2.13.1
```

Comment:
Why not even 2.13.2? #reckless?

Reply:
Nice catch. Main pom.xml was also using 2.13.1. Fixed in ed5f7c707.